go/vt/sqlparser/ast.go

```diff
@@ -588,15 +571,18 @@ func (*AlterTable) iStatement()        {}
 func (*AlterVschema) iStatement()      {}
 func (*DropTable) iStatement()         {}
 func (*DropView) iStatement()          {}
-
-func (*DDL) iDDLStatement()         {}
-func (*CreateIndex) iDDLStatement() {}
-func (*CreateView) iDDLStatement()  {}
-func (*AlterView) iDDLStatement()   {}
-func (*CreateTable) iDDLStatement() {}
-func (*DropTable) iDDLStatement()   {}
-func (*DropView) iDDLStatement()    {}
-func (*AlterTable) iDDLStatement()  {}
+func (*TruncateTable) iStatement()     {}
+func (*RenameTable) iStatement()       {}
+
+func (*Flush) iDDLStatement()         {}
```

Comment:
flush is other administrative statement. I think we should remove it from ddl statement.
https://dev.mysql.com/doc/refman/8.0/en/flush.html

Reply:
Done!